### ISSUE TITLE
Create a version that only updates code on submit

### ISSUE THREAD

**Issue by Wildebrew** [lloydi/markup-de-crapulator#16]
This is my final request from a screen reader user perspective.
Currently the code is updated live as you check/uncheck the various options. When running a combination of Chrome and screen readers this often causes unpredictable browser behavior, focus jumping all over the place, having to relocate the last changes you made etc.
If there was a version where you can make your changes and only process them by clicking "submit" the screen reader behavior would be much easier to manage.
Basically this version would only do its thing when you click "submit".
You could call this "basic mode" or something. I think the regular behavior is probably better for many sighed users, seeing the changed and the results in real time, so I think this would be best served as an alternative.
It's probably easy to implement, though I haven't looked at the underlying code. ;)

**Comment by lloydi** [comment]
Again, do-able. However there should not be any focus jumping around so if you can let me know how that occurs I will check. I don't move focus on purpose because I know that is VERY BAD for this sort of thing :)

**Comment by lloydi** [comment]
Now fixed - check out the new preferences section, @Wildebrew
